modules/virt-about-containerdisks-for-vms.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+[id="virt-about-containerdisks-for-vms_{context}"]
+= About using containerDisk volumes for virtual machines
+
+A container disk is a virtual machine image stored as a container in an
+image registry. You can attach a container disk as an ephemeral containerDisk volume to
+a virtual machine by using the virtual machine wizard or by adding it to a
+virtual machine manifest.
+
+Container disks are not limited to a single virtual machine and can be used
+to create large numbers of virtual machine clones.
+
+When a virtual machine with a containerDisk volume starts, the container
+is pulled from the registry and hosted on the node that is hosting the virtual machine.
+
+[NOTE]
+====
+A containerDisk volume is ephemeral. It is discarded when
+the virtual machine is stopped, restarted, or deleted. A containerDisk
+is useful for read-only filesystems such as CD-ROMs or for disposable
+virtual machines.
```

Comment:
Maybe we can use an even harsher language.

Using it for r/w filesystems would cause data to be temporarily written on the local storage of the worker node that runs the VM, and would slow down live migration of the VM in case the node goes to maintenance.